app/assets/javascripts/subjects.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
We probably don't need this file. The drawback of using scaffold is we create unnecessary asset files. I recently learned there's an option to not generate asset files when scaffolding. For now, what do you think about removing this?